### PR TITLE
okn filters

### PR DESCRIPTION
**Pull request by hiba9201**
- filters for okn

### CODE REVIEW

File: components/UI/Map/Point/Point.module.css [hunk 1]
```diff
@@ -14,24 +14,33 @@
...
     box-shadow: 0 0 0 5px;
     z-index: 9999 !important;
 }
 
-.point.point_size_m {
+.point_size_m {
+    box-shadow: 0 0 0 4px currentColor;
+}
+
+.point_size_m:hover {
```
Comment (a-chabin) [inline components/UI/Map/Point/Point.module.css:27]: Let's use nesting for pseudo-selectors `&:hover` and elsewhere in the code :)

File: components/UI/Map/Point/Point.tsx [hunk 2]
```diff
@@ -41,10 +45,26 @@ function PointComponent({
...
         });
 
-        return `<div class="${className}" style="${style}"></div>`;
-    }, [isOpen, size, color, sizeNumber]);
+        if (preview) {
+            return `<img
+                src="${preview}"
+                class="${className}"
+                style="${style}"
+                alt="Превью объекта"
```
Comment (a-chabin) [inline components/UI/Map/Point/Point.tsx:56]: Let's remove the same `alt=""` and add more useful information. For example, with the name of the object.

Or until we delete it altogether.

Comment (a-chabin) [reply components/UI/Map/Point/Point.tsx:56]: It is ideal to wrap everything in buttons with the `aria-haspopup` attribute

File: components/Model/OKN/Filter/Okn/OknFilter.tsx [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                                onClick={onAreaChange(type)}
+                            >
+                                {type}
+                                <span className={styles.OknFilter__objectsCount}>{count}</span>
+                                <br />
+                                <small className={styles.OknFilter__checkboxDescription}>
+                                    {AREA_CONFIG[type].description}
+                                </small>
+                            </Checkbox>
+                            <div className={styles.OknFilter__checkboxContent} />
```
Comment (a-chabin) [inline components/Model/OKN/Filter/Okn/OknFilter.tsx:151]: Let's remove empty `checkboxContent`

**Comment by a-chabin** [comment]
Check my comments and we're ready to merge :)

File: components/UI/Map/Point/Point.tsx (hunk 2 above)
Comment (hiba9201) [reply components/UI/Map/Point/Point.tsx:56]: let's do it in separate task, i have a couple of questions about roles

**Pull request merged**